### PR TITLE
Remove narrow no-break space

### LINKED ISSUE #154: command to retrieve ids fails in codespaces. 01-workshop-environment-setup
fails
```
export LOG_ANALYTICS_RESOURCE_ID=$(az monitor log-analytics workspace show \ 
    --resource-group ${RESOURCE_GROUP} \ 
    --workspace-name ${LOG_ANALYTICS_WORKSPACE} | jq -r '.id')  
```
works
`export LOG_ANALYTICS_RESOURCE_ID=$(az monitor log-analytics workspace show --resource-group ${RESOURCE_GROUP}  --workspace-name ${LOG_ANALYTICS_WORKSPACE} | jq -r '.id')  `

### PR DESCRIPTION
**Pull request by dingmeng-xue**
narrow no-break space 0x202f breaks code in codespace. Replace them with normal whitespace character.

### CODE REVIEW

File: azure-spring-apps-enterprise/workshops/azure-spring-apps-enterprise/intermediate/01-workshop-environment-setup/README.md [hunk 1]
```diff
@@ -225,50 +225,45 @@ export MSYS_NO_PATHCONV=1
...
+           } 
+         }, 
+         { 
+            "category": "SystemLogs", 
+            "enabled": true, 
+            "retentionPolicy": { 
+             "enabled": false, 
+             "days": 0 
+            } 
+          },
```
Comment (jiec-msft) [inline azure-spring-apps-enterprise/workshops/azure-spring-apps-enterprise/intermediate/01-workshop-environment-setup/README.md]: nit: we can format this json structure to beautify the indents and alignments.

File: azure-spring-apps-enterprise/workshops/azure-spring-apps-enterprise/intermediate/01-workshop-environment-setup/README.md [hunk 2]
```diff
@@ -225,50 +225,45 @@ export MSYS_NO_PATHCONV=1
...
+         { 
+            "category": "IngressLogs", 
+            "enabled": true, 
+            "retentionPolicy": { 
+              "enabled": false, 
+              "days": 0 
+             } 
+           } 
+       ]' \
+       --metrics '[
```
Comment (jiec-msft) [inline azure-spring-apps-enterprise/workshops/azure-spring-apps-enterprise/intermediate/01-workshop-environment-setup/README.md]: nit: `--metrics` should align with `--logs`

Comment (dingmeng-xue) [reply azure-spring-apps-enterprise/workshops/azure-spring-apps-enterprise/intermediate/01-workshop-environment-setup/README.md]: I cannot follow it. Please explain further.

File: azure-spring-apps-enterprise/workshops/azure-spring-apps-enterprise/intermediate/01-workshop-environment-setup/README.md (hunk 1 above)
Comment (dingmeng-xue) [reply azure-spring-apps-enterprise/workshops/azure-spring-apps-enterprise/intermediate/01-workshop-environment-setup/README.md]: Formatted.

File: azure-spring-apps-enterprise/workshops/azure-spring-apps-enterprise/intermediate/01-workshop-environment-setup/README.md (hunk 2 above)
Comment (jiec-msft) [reply azure-spring-apps-enterprise/workshops/azure-spring-apps-enterprise/intermediate/01-workshop-environment-setup/README.md]: I see it's ready updated when you update for the format 👍 
BTW, I was trying to say that `--metrics` is indented with 6 spaces, whereas `--logs` is indented with 4 spaces.

**Pull request merged**